logstash-core/benchmarks/build.gradle

```diff
@@ -28,7 +28,6 @@ String jrubyVersion = versionMap['jruby']['version']
 
 repositories {
   mavenCentral()
-  jcenter()
```

Comment:
Nit: Should this be in https://github.com/elastic/logstash/pull/13184?

Reply:
Yes, absolutely! Thank's for pointing it